discussion/views.py

```diff
@@ -185,6 +188,17 @@ def create_attachments(request):
     # The user is uploading one or more files.
     ret = { }
     for fn in request.FILES:
+        # Validate before attachment object creation
+        uploaded_file = request.FILES[fn]
+
+        # 2.5MB
+        if sys.getsizeof(uploaded_file) >= DATA_UPLOAD_MAX_MEMORY_SIZE:
```

Comment:
Should be `>` rather than `>=`, no?

(Not that the `==` case will ever likely happen.)

Reply:
I think 2.5MB is the suspect limit for the request so wanted to avoid the unlikely but am not married to it. I think this will be addressed in issue 1077/1078